packages/x-date-pickers/src/AdapterMomentHijri/AdapterMomentHijri.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import moment from 'moment';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { AdapterMomentHijri } from '@mui/x-date-pickers/AdapterMomentHijri';
+import { screen } from '@mui/monorepo/test/utils/createRenderer';
+import {
+  createPickerRenderer,
+  expectInputPlaceholder,
+  expectInputValue,
+} from 'test/utils/pickers-utils';
+import { describeHijriAdapter } from '@mui/x-date-pickers/tests/describeHijriAdapter';
+import 'moment/locale/ar';
+import { AdapterFormats } from '@mui/x-date-pickers';
+import { expect } from 'chai';
+
+describe('<AdapterMomentHijri />', () => {
+  describeHijriAdapter(AdapterMomentHijri, {
+    before: () => {
+      moment.locale('ar-SA');
+    },
+    after: () => {
+      moment.locale('en');
+    },
+  });
+
+  describe('Adapter localization', () => {
+    it('Formatting', () => {
+      const adapter = new AdapterMomentHijri();
+
+      const expectDate = (format: keyof AdapterFormats, expectedWithFaIR: string) => {
+        const date = adapter.date('2020-01-01T23:44:00.000Z')!;
+
+        expect(adapter.format(date, format)).to.equal(expectedWithFaIR);
```

Comment:
Isn't it arSA for this adapter? 🤔 
```suggestion
      const expectDate = (format: keyof AdapterFormats, expectedWithArSA: string) => {
        const date = adapter.date('2020-01-01T23:44:00.000Z')!;

        expect(adapter.format(date, format)).to.equal(expectedWithArSA);
```